driver/tests/swe_roe/ex2b_checkpoint.yaml

```diff
@@ -0,0 +1,88 @@
+# This test input corresponds to toy-problems/swe_roe/ex2b.c.
```

Comment:
How about updating the `ex2b.yaml` to include the `checkpoint` block instead of this new file?

Reply:
Sure, I can consolidate the tests a bit.